app/models/declaration.rb

```diff
@@ -1,9 +1,14 @@
 class Declaration < ApplicationRecord
+  BILLABLE_STATES = %w[submitted eligible payable paid].freeze
```

Comment:
hmm not sure about including submitted in billable, as it's not technically a billable state 🤔

Reply:
in ECF there's changeable, maybe we include those and then we can merge scopes? let me know your thoughts

Reply:
Best option is to revert both commits: constant & scopes?
cc/ @cwrw @ethax-ross 

We could extract them based on actual code duplications?

Reply:
Discussed in person (@cwrw @ethax-ross), we have decided to rely on:

```ruby
scope :billable, -> { where(state: %w[eligible payable paid]) }
scope :changeable, -> { where(state: %w[eligible submitted]) }
scope :billable_or_changeable, -> { billable.or(changeable) }
```

 See https://github.com/DFE-Digital/npq-registration/pull/1435/commits/1b12f06c9b373614830cc3b9088e4939708437cb